src/state/state_api.rs

```diff
@@ -91,8 +93,8 @@ pub trait State {
     fn set_sierra_program(
         &mut self,
         compiled_class_hash: &Felt252,
-        sierra_program: Vec<BigUintAsHex>,
-    ) -> Result<(), StateError>;
+        sierra_program: Arc<(SierraProgram, ContractEntryPoints)>,
```

Comment:
same here

Reply:
Same as before, we don't have this method anymore.